src/main/java/org/opensearch/performanceanalyzer/reader/MetricsEmitter.java

```diff
@@ -303,11 +314,8 @@ private static void checkInvalidData(
 
         Condition condition = DSL.trueCondition();
```

Comment:
Since I didn't change it I wouldn't want to risk removing this and causing unexpected errors. Will let it be as is. I am not sure about the syntax for the sql query where this is condition is being passed.